sdk/eventgrid/azure-eventgrid/README.md

```diff
@@ -121,6 +136,32 @@ credential = AzureKeyCredential(key)
 client = EventGridPublisherClient(topic_hostname, credential)
 
 client.send(event)
+# Note: Events may also be sent as a list, e.g: client.send([event])
+```
+
+### Send to a Domain
+
+This example illustrates sending to a Domain, whereas the earlier samples sent to a Topic.
```

Comment:
This may add a little more clarity?
```suggestion
This example illustrates sending to a Domain Topic, whereas the earlier samples sent to a custom Topic.
```

Reply:
why custom?

Reply:
I thought it would be more consistent with the way we name the samples. For example, [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_cloud_events_to_custom_topic_sample.py#L32) or [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventgrid/azure-eventgrid/samples/publish_samples/publish_event_grid_events_to_custom_topic_sample.py). It may also make more sense to rename these samples to `...events_to_topic` instead of `events_to_custom_topic`?